recipes/hg-git/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "hg-git" %}
+{% set version = "0.8.13" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 0f0a774e2f94e305ab69d13a5ea1e7e440b4b03486fc1a30784715fcd2ee6290
+
+build:
+  number: 0
+  skip: True  # [py3k]
```

Comment:
I think this recipe can be `noarch`. Please remove the `skip` line, add `noarch: python`, and in the `host` and `run` sections use `python <3`.